act/utils/data_utils.py

```diff
@@ -1410,3 +1410,112 @@ def calculate_percentages(ds, fields, time=None, time_slice=None, threshold=None
         percentages[i] = j
     ds_percent.close()
     return percentages
+
+
+def convert_2d_to_1d(ds_object, parse=None, variables=None, keep_name_if_one=False,
```

Comment:
@ajsockol Can we rename ds_object to ds in this new code to be consistent with other functions in ACT.